src/redux/sagas/utils/index.ts

```diff
@@ -1,7 +1,6 @@
 export { default as getColonyManager } from './getColonyManager';
 export { default as getNetworkClient } from './getNetworkClient';
 export { default as getGasPrices } from './getGasPrices';
-export { default as createUserWithSecondAttempt } from './createUserWithSecondAttempt';
```

Comment:
I am presuming this is no longer necessary, but I don't have the context for why we have it. Something to do with IPFS?